twisted/internet/test/test_inotify.py

```diff
@@ -128,8 +127,7 @@ def test_closeWrite(self):
         callback.
         """
         def operation(path):
-            fObj = path.open("w")
-            fObj.close()
+            path.open("w").close()
```

Comment:
Whoops, should have looked at the commit messages :( This is good!

On Thursday, July 21, 2016, Ville Skyttä <notifications@github.com> wrote:

> In twisted/internet/test/test_inotify.py
> <https://github.com/twisted/twisted/pull/377#discussion_r71745461>:
>
> > @@ -128,8 +127,7 @@ def test_closeWrite(self):
> >          callback.
> >          """
> >          def operation(path):
> > -            fObj = path.open("w")
> > -            fObj.close()
> > +            path.open("w").close()
>
> @markrwilliams <https://github.com/markrwilliams> like the message for
> the commit which this change is part of says, "Make trivial open/close
> operations oneliners for easier greppability". So it's easier to grep for
> example open invocations without def, with, close etc when looking for
> files that may not be explicitly closed.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/twisted/twisted/pull/377/files/13eb01b44bb279f46042847c205f55f067f27ea3#r71745461>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAq-9wf9jVE2dNjIcq1_Mx-R1mZnkKtYks5qX6bSgaJpZM4JRmzW>
> .
>